__tests__/app.test.js

```diff
@@ -525,4 +525,79 @@ describe('GET /api/users', () => {
     });
 });
 
+describe.only('GET /api/articles(topic query)', () => {
+    test('should return an article array with status 200 if successful', () => {
+        return request(app)
+        .get('/api/articles')
+        .expect(200)
+        .then((response)=>{
+            const article = response.body
+            expect(Array.isArray(article)).toBe(true)
+        })
+    });
+    test('should return an article array with all the article properties', () => {
+        return request(app)
+        .get('/api/articles')
+        .expect(200)
+        .then((response)=>{
+            const articles = response.body
```

Comment:
As we are using a forEach here - if the array is empty then this test will pass by default as none of the expect statements will be called. A test with no expect statements will pass by default. We should first check that there is something in the array (check its length)